packages/expo-dev-menu/app/components/Onboarding.tsx

```diff
@@ -0,0 +1,70 @@
+import { Heading, View, Text, Spacer, Button } from 'expo-dev-client-components';
+import * as React from 'react';
+import { Platform, StyleSheet } from 'react-native';
+
+import { useMenuPreferences } from '../hooks/useMenuPreferences';
+import { GestureHandlerTouchableWrapper } from './GestureHandlerTouchableWrapper';
+
+const deviceMessage = Platform.select({
+  ios: `Since this is your first time opening Expo Go, we wanted to show you this menu and let you know that you can shake your device or long press anywhere on the screen with three fingers to get back to it at any time.`,
+  android: `Since this is your first time opening Expo Go, we wanted to show you this menu and let you know that you can shake your device to get back to it at any time.`,
```

Comment:
three finger long press should work on Android devices as well -- at least, it does on mine (Pixel 2)